pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Issue
+<!-- Example:
+Error when importing the sdk
+-->
+
+## Description
+<!-- Explain what this Pull Request should do -->
+<!-- Example:
+This PR fixes a bug in our elastic search order query.
+-->
+This PR
```

Comment:
What do you think about moving this file inside the `.github` folder, as it organizes better in my opinion.

Reply:
Thanks for the suggestion.